communication/marketing-team/social-guidelines.md

```diff
@@ -24,19 +24,42 @@ Kubernetes Organization Members with access to account are:
 ## Themes/Topics
 
 - Contributor/SIG blog posts
-- Events
-- KEPs
+- Contributor events
+- KEPs*
+  - see [#KEPs] section for further guidance
 - Process changes
 - Release info, code freeze, et al
 - Accomplishments (reaching 100,000 GitHub stars for example)
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
+- Steering Committee Announcements
+
+## KEPs
+([Kubernetes Enhancement Proposals])
+
+### Ground Rules  
+- Social Media team can not proactively look for KEPs to amplify.
+- It's not ok to tweet about a KEP that is going to status: implementable until it's been merged.
+  - a good example is user feedback for features that are alpha
+- KEP metadata is found in the kubernetes/enhancements repo only and not on subsequent PRs.
+
+### Process  
+1. KEP requests must come from the author in the metadata; author will apply the area/contributor-comms label
+2. All tweets and content of tweet(s) must be approved explicitly on the KEP in GitHub by author/reviewer/approver listed in the metadata; the author is required and at least one party from the reviewer and/or approver category in the metadata
+3. If there is no meta data on the KEP, you can not amplify it on any social or communication channels
+4. To guide the discussion effectively, it's recommended to link to a discuss.kubernetes.io topic for the KEP unless asked to link to a GitHub issue by the Author.
```

Comment:
I find this section very confusing. Part of that is a lack of total understanding of the KEP process. I would very much like to simplify this language because, right now, as written I'm not going to feel 100% comfortable tweeting a KEP. If the intent behind the message is don't tweet KEPs and when asked via tag tweet a link to discuss.k.io can we just say that.

Reply:
since the kep subproject went under sig-arch now, what if sig-chairs for arch and subproject owners for the kep project will bring the tweet to us? is that easier on the team @chris-short and wdyt @johnbelamaric @justaugustus @dims @derekwaynecarr @mrbobbytables @jeremyrickard 

i think these will be rare/not frequent requests and maybe thats something you all can also discuss in a sig-arch meeting when the occasion arises?

Reply:
The Enhancements subproject deals with process and machinery for KEPs. Definitely do NOT want to be in the path for managing specific KEPs. "Managing" includes shepherding, delivery, marketing and all that jazz. That's up to the owning SIGs, KEP authors, and release teams.

Reply:
@johnbelamaric could we say that SIG Chairs and/or Tech Leads could bring us the tweet?